cmd/cosign/cli/verify.go

```diff
@@ -50,6 +51,7 @@ func applyVerifyFlags(cmd *VerifyCommand, flagset *flag.FlagSet) {
 	flagset.StringVar(&cmd.KeyRef, "key", "", "path to the public key file, URL, KMS URI or Kubernetes Secret")
 	flagset.BoolVar(&cmd.Sk, "sk", false, "whether to use a hardware security key")
 	flagset.StringVar(&cmd.Slot, "slot", "", "security key slot to use for generated key (default: signature) (authentication|signature|card-authentication|key-management)")
+	flagset.StringVar(&cmd.RekorURL, "rekor-url", "https://rekor.sigstore.dev", "address of rekor STL server")
```

Comment:
`s/rekor-url/rekor-server/`

Reply:
I honestly prefer url, as it is a url. Are we using rekor-server anywhere else in cosign?

Reply:
`rekor-server` was used everywhere else in this PR except here, so I assumed that's what you intended here as well.

Reply:
IMHO we should probably `s/rekor-server/rekor-url/`

Reply:
+1 to using `rekor-url`.

Reply:
I am bit lost here, there is no current 'rekor-server' vars

https://github.com/sigstore/cosign/search?q=rekor-server

The only rekorServer was a const used in a helper function and this PR removes that

https://github.com/sigstore/cosign/blob/7393e9667083744d13686fbe2ff0d511f5fe5063/cmd/cosign/cli/util.go#L34-L54

![image](https://user-images.githubusercontent.com/7058938/126601629-9d086368-4880-4917-a55f-07b6abb59326.png)